mkdocs.yml

```
@@ -5,565 +5,566 @@ repo_url: https://github.com/ezsystems/developer-documentation
+            - Binary and Media download: guide/file_management/binary_media_download.md
+            - File URL handling: guide/file_management/handling_file_url.md
+        - Page:
+            - Page blocks: guide/page/page_blocks.md
+            - Page block attributes: guide/page/page_block_attributes.md
+            - Page block validators: guide/page/page_block_validators.md
+            - Create custom Page block: guide/page/create_custom_page_block.md
+        - Forms:
+            - Forms: guide/form_builder/forms.md
+            - Form API: api/public_php_api_managing_forms.md
+            - Create custom Form field: guide/form_builder/create_custom_form_field.md
+            - Customize email notifications: guide/form_builder/customize_email_notifications.md
+        - Workflow:
+            - Workflow: guide/workflow/workflow.md
+            - Add custom workflow action: guide/workflow/add_custom_workflow_action.md
+        - URL management:
+            - URL management: guide/url_management.md
+            - URL API: api/public_php_api_url_service.md   
+        - User-generated content: guide/user_generated_content.md
+        - Taxonomy:
+            - Taxonomy: guide/taxonomy/taxonomy.md
+        - Notifications: guide/sending_notifications.md
+        - Content API:
+            - Browsing content: api/public_php_api_browsing.md
+            - Creating content: api/public_php_api_creating_content.md
+            - Managing content: api/public_php_api_managing_content.md
+        - Data migration:
+            - Data migration: guide/data_migration/data_migration.md
+            - Importing data: guide/data_migration/importing_data.md
+            - Exporting data: guide/data_migration/exporting_data.md
+            - Managing migrations: guide/data_migration/migration_management.md
+            - Data migration actions: guide/data_migration/data_migration_actions.md
+            - Create data migration action: guide/data_migration/create_migration_action.md
+            - Create data migration step: guide/data_migration/create_migration_step.md
+            - Add data migration matcher: guide/data_migration/add_data_migration_matcher.md
+            - Data migration API: api/public_php_api_managing_migrations.md
+        - Field Types:
+            - Field Types: api/field_type_api.md
+            - Type and Value: api/field_type_type_and_value.md
+            - Form and template: api/field_type_form_and_template.md
+            - Storage: api/field_type_storage.md
+            - Validation: api/field_type_validation.md
+            - Searching: api/field_type_search.md
+            - Create custom generic Field Type: api/field_type/create_custom_generic_field_type.md
+            - Create custom Field Type comparison: api/field_type/create_custom_field_type_comparison.md
+            - Field Type reference: api/field_type_reference.md
+            - Field Type reference:
```

```suggestion         
            - Field Type reference:
               - Field Type reference: api/field_type_reference.md
```
repetition on the same level